### ISSUE TITLE
Bug: `enfroce-consistent-variable-syntax` reports incorrect variable definition syntax if custom prefix is used

### ISSUE THREAD

**Issue by andreww2012** [schoero/eslint-plugin-better-tailwindcss#187]
If Tailwind custom prefix is used, the following code will be reported by `enfroce-consistent-variable-syntax`:

```html
<div class="tw:[--green:#00ff00]">
  ...
</div>
```

Reproduction: https://stackblitz.com/edit/vitejs-vite-evc7f4ln
